mlrun/model_monitoring/stream_processing_fs.py

```diff
@@ -621,10 +618,11 @@ def do(self, full_event):
         # In case this process fails, resume state from existing record
         self.resume_state(endpoint_id)
 
-        # Handle errors coming from stream
-        found_errors = self.handle_errors(endpoint_id, event)
-        if found_errors:
-            return None
+        # If error key has been found in the current event,
+        # increase the error counter by 1 and raise the error description
+        if "error" in event:
+            self.error_count[endpoint_id] += 1
+            raise mlrun.errors.MLRunInvalidArgumentError(f"{event['error']}")
```

Comment:
I guess this is a bit better
```suggestion
        error = event.get("error")
        if error:
            self.error_count[endpoint_id] += 1
            raise mlrun.errors.MLRunInvalidArgumentError(str(error))
```